tendermint-rs/src/secret_connection/mod.rs

```diff
@@ -123,34 +111,28 @@ impl<IoHandler: Read + Write + Send + Sync> SecretConnection<IoHandler> {
     }
 
     /// Unseal (i.e. decrypt) AEAD authenticated data
-    fn open(&self, authtext: &[u8], ciphertext: &[u8], out: &mut [u8]) -> Result<usize, Error> {
+    fn open(&self, ciphertext: &[u8], out: &mut [u8]) -> Result<usize, Error> {
```

Comment:
⚠️**ATTENTION** ⚠️

While the rest of the changes are mostly boilerplate and 2018 edition churn, the ones below are actually important.

This PR vicariously transitively upgrades the `yubihsm` crate, and with that, `ring` (guess we're on 4 levels of transitive dependencies at this point). Namely, it now uses `ring` v0.14.

This release made a number of API changes, and to accommodate them I ended up rewriting much of this method. The tests pass, but it's nontrivial changes to cryptography code, and could definitely do with some more thorough review.

Reply:
The changes look good to me, too.